autopkg/autopkg_tools.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, path):
         self.error = False
         self.results = {}
         self.updated = False
+        self.verified = False
```

Comment:
Can this to be `None`? This will allow us to process recipes without trust info (in conjunction with a `--verify-trust-info` flag, suggested below.